src/model/APNSResponse.php

```diff
@@ -62,7 +68,7 @@ function shouldUnsubscribeDevice(): bool {
 	}
 
 	function shouldRetry(): bool {
-		return $this->status_code === 429 || $this->isServerError();
+		return $this->status_code === 429 || $this->isServerError() || 0 === $this->status_code;
```

Comment:
Nitpick, consistent positioning of actual and expected:

```suggestion
		return $this->status_code === 429 || $this->isServerError() || $this->status_code === 0;
```

Reply:
Yeah makes sense! The inconsistent one is the WP.com style – I'm internalizing it already 😅